### PR TITLE
fix docs link (ar)

### PR DESCRIPTION
**Pull request by Arhell**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ❌
| Fixed issues  | https://github.com/yiisoft/yii2/issues/18647

use google translate

### CODE REVIEW

File: docs/guide-ar/intro-yii.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Yii هو إطار عام لبرمجة الويب ، مما يعني أنه يم
 </p>
 
 <p dir="rtl">
-ال Yii إطار عمل صمم من قبل فريق برمجي متكامل، فهو ليس مجرد عمل فردي ،  بل يتكون من فريق تطوير أساسي وقوي(http://www.yiiframework.com/team/) ، بالإضافة إلى منتدى كبير
+ال Yii إطار عمل صمم من قبل فريق برمجي متكامل، فهو ليس مجرد عمل فردي ،  بل يتكون من فريق تطوير أساسي <a href="https://www.yiiframework.com/team">فريق تطوير أساسي وقوي</a> ، بالإضافة إلى منتدى كبير
```
Comment (samdark) [inline docs/guide-ar/intro-yii.md:31]: I'm not sure that's correct.

Comment (samdark) [reply docs/guide-ar/intro-yii.md:31]: @aneeshikmat would you please check it?

Comment (aneeshikmat) [reply docs/guide-ar/intro-yii.md:31]: Sure, I will check it, thank you

Comment (aneeshikmat) [reply docs/guide-ar/intro-yii.md:31]: @Arhell Thank you for this update, I agree with your improvement to replace a printed link to be as a hyper text link, but the text now is duplicated, you need to remove "فريق تطوير أساسي"  text written before "فريق تطوير أساسي وقوي".

@samdark we need this fix and all will be fine...

Comment (samdark) [reply docs/guide-ar/intro-yii.md:31]: We're terrible with Arabic :) Maybe you have some time for a pull request?

Comment (samdark) [reply docs/guide-ar/intro-yii.md]: Is that alright now?

**Comment by Arhell** [comment]
@aneeshikmat  done, check please

**Review by aneeshikmat — APPROVED** [review]

**Comment by aneeshikmat** [comment]
> @aneeshikmat done, check please

Thank you!

File: docs/guide-ar/intro-yii.md (hunk 1 above)
Comment (aneeshikmat) [reply docs/guide-ar/intro-yii.md]: Its right now, yes

**Comment by bizley** [comment]
Thank you, guys!

**Pull request merged**